Adding typescript template (#13)
apollographql/rover-init-starters: start-with-typescript/README.md

# Rover init Start with Typescript template

> This template can be used to quickly create an [Apollo Federation] subgraph with the [@apollo/subgraph] and [@apollo/server] packages. You can use this template from [Rover](https://www.apollographql.com/docs/rover/commands/template/) with `rover init` and selecting the "start with Typescript" journey.

Reviewer: Looks like the links are missing URLs. Also a touch of copyediting:
- Prefer [active voice ](https://developers.google.com/style/voice#:~:text=In%20general%2C%20use%20active%20voice,clear%20who's%20performing%20the%20action.)
- Avoid words like ["quickly", "easily", etc.](https://developers.google.com/style/word-list#quick)

```suggestion
This template helps create an [Apollo Federation](https://www.apollographql.com/docs/graphos/schema-design/federated-schemas/federation) subgraph with [Apollo Server](https://www.apollographql.com/docs/apollo-server) and the [@apollo/subgraph](https://www.apollographql.com/docs/apollo-server/using-federation/api/apollo-subgraph) package. You can use this template with the [`rover init`](https://www.apollographql.com/docs/rover/commands/init) command by selecting **Start with Typescript** in the command's wizard.
```

## What's Included

- A basic, [Apollo Federation] subgraph with simple examples for queries, entities, and mutations. You can run this subgraph with `npm start`.
- [nodemon] is setup for `npm run dev` for a hot-reloading development environment.
- Example tests in the `src/__tests__` directory. You can run these tests with `npm run test`.

> - [GraphQL Code Generator] pre-configured as a `build` step.

Reviewer: Seeing this again, so it's probably known ;), but looks like URLs for links are missing

> ## Next Steps

Reviewer: ```suggestion
## Next steps
```

It feels like this section includes both a "how to get started" as well as additional resources--can we split those out please?
